<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPurge</class>
 <widget class="QDialog" name="DialogPurge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Purge messages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_queuename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Queue purge</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 2px solid red;
padding:   0.5em;
color: black;
background-color: #ffdddd;</string>
     </property>
     <property name="text">
      <string>Please enter a valid number &gt;= 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_count">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
     padding: 0 3px;
	left: 1em;
 }</string>
     </property>
     <property name="title">
      <string>Select whch messages to purge</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radioButton_top">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>271</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Purge the top message in the queue</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_number">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>271</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Purge the top             messages</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_all">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>321</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Purge all messages in the queue</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPurge</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPurge</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>radioButton_number</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
